miner/worker.go

```diff
@@ -592,7 +597,11 @@ func (w *worker) mainLoop() {
 				}
 				txset := types.NewTransactionsByPriceAndNonce(w.current.signer, txs, w.current.header.BaseFee)
 				tcount := w.current.tcount
-				w.commitTransactions(w.current, txset, nil)
+				err := w.commitTransactions(w.current, txset, nil)
```

Comment:
No strong opinions from my side.
Usually I'd avoid discarding errors to avoid refactoring bugs in the future but you are correct that it's not needed right now, same for `generateWork`.